src/managers/s3.py

```diff
@@ -28,6 +29,41 @@ def session(self):
             aws_secret_access_key=self.config.secret_key,
         )
 
+    def get_or_create_bucket(self, ca_file_name: str) -> bool:
+        """Create bucket if it does not exists."""
+        s3 = boto3.resource(
```

Comment:
Done in 636ba804b2809326238528aa7744550134588683. I had to implement the `wait_unit_exists` method using the low-level s3 client instead of letting the resource object handle it, but it's not that bad